tools/packaging/kernel/build-kernel.sh

```diff
@@ -5,7 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 description="
-Description: This script is the *ONLY* to build a kernel for development.
+Description: This script is the *ONLY* way to build a kernel for development.
```

Comment:
While this fixes grammar, the description gives me pause.

@jcvenegas can you clarify the intention w/ this description?

Reply:
@dcmiddle @egernst  long time ago, but what I wanted to say was the was  just for development purposes, for someone integrating in their stack or OS was preferred to use packages. But today there is not packages at all.  Today a  better description may be: 
"Automation script to build a guest kernel for Kata"?

Reply:
Maybe just remove the description? There's already an overview
```
Overview:

          Build a kernel for Kata Containers
```

Reply:
I rebased in a change to remove the Description.
I think all PR feedback and static tests are now satisfied.